### PR TITLE
Provide examples to query for SVs and cases in readme file

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds an example on how to 
- Send a query the svs endpoint
- Send a query to the cases endpoint to get n_cases with SNVs and n_cases with SVs
- send a query to cases endpoint to get info on a specific case

### How to test:
- Follow instructions added on readme

### Review:
- [ ] Code approved by
- [x] Tests executed by CR
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Query for SVs:

image

Query n_cases:

image


Query one case:
image

**Comment by moonso** [comment]
Hi! Did you try the generated docs? Run the server and go to host:port/docs, it is super nice interface to test the REST api

**Comment by northwestwitch** [comment]
> Hi! Did you try the generated docs? Run the server and go to host:port/docs, it is super nice interface to test the REST api

Yes, I tried it and it's nice, but they don't provide an example with real parameters, just what the endpoint accepts and returns:

image

And it doesn't say how then GET request should be shaped (the actual string that you type either in the browser or in curl)

File: README.md [hunk 1]
```diff
@@ -47,7 +47,11 @@ Head over to `http://127.0.0.1:8000/docs`
 
 1. `docker-compose up`
 1. Port `80` is mapped to `9000` in the docker compose file so go to `http://127.0.0.1:9000/docs`
-1. Fetch a variant with `curl -X GET "http://127.0.0.1:9000/variants/1_880086_T_C" -H  "accept: application/json"`
```
Comment (dnil) [inline README.md:50]: Any particular reason to kill the accept: json? Either is fine in my book..

File: README.md [hunk 2]
```diff
@@ -47,7 +47,11 @@ Head over to `http://127.0.0.1:8000/docs`
...
+1. Request example to fetch a single-nucleotide (SNV) variant: `curl -X GET "http://127.0.0.1:9000/variants/1_880086_T_C"`
+1. Request example to fetch a structural variant (SV): `curl -X GET "http://127.0.0.1:9000/svs/?chrom=1&end_chrom=1&pos=7889972&end=7890026&sv_type=DEL"`
```
Comment (dnil) [inline README.md:51]: 💯

**Review by dnil — APPROVED** [review]

**Comment by dnil** [comment]
Also, I would rather say its a patch, for just extending docs?

File: README.md (hunk 1 above)
Comment (northwestwitch) [reply README.md:50]: Perhaps it works without?

**Comment by northwestwitch** [comment]
> Also, I would rather say its a patch, for just extending docs? Changing that, but protest if you disagree obviously!

Doesn't really matter, patch is good!

**Comment by northwestwitch** [comment]
Merged as a patch, thanks @dnil!

**Pull request merged**